tsconfig.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
As discussed in meeting, it's better for this file to only include what you are manually setting. Remove the defaults.